settings.gradle

```diff
@@ -18,6 +19,28 @@ dependencyResolutionManagement {
     }
 }
 
+dependencyResolutionManagement {
```

Comment:
i guess, instead node plugin should be provided directly in every module

Reply:
Do you mean we should remove node plugin from `typescript.gradle` and apply it in the `build.gradle` in each library instead to get rid of those hacks with `dependencyResolutionManagement`?

Reply:
Made a bit of adjustments, but the [core problem](https://github.com/node-gradle/gradle-node-plugin/blob/master/docs/faq.md#is-this-plugin-compatible-with-centralized-repositories-declaration) hasn't gone. If the node download option is set:
```gradle
node {
    download = true
    version = '16.15.0'
}
```
We either get problems with adding a repository for the Node to download, or the newly added repository overrides existing one (default mode or explicitly setting `RepositoriesMode.PREFER_PROJECT` mode). I don't see a way to remove the `ivy` repo from `dependencyResolutionManagement` right now. Maybe you can take a look.